Cartfile

```diff
@@ -1,5 +1,5 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" "16.0.0"
```

Comment:
I wonder whether it really worked before. Carthage emits such error in case if we use `"` for binary dependencies:

```
Parse error: binary dependencies cannot have a git reference for the version specifier in line: binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" "16.0.0"
```

Reply:
This was broken in #3221, probably due to hand-editing the Cartfile as if it were a Cartfile.resolved, but we didn’t catch the issue somehow.

Reply:
By the looks of it yes. Reverted to previous version and changed to `16.2.0`.